src/Components/modal/DeleteModal.tsx

```diff
@@ -72,16 +72,17 @@ const DeleteButton = styled.button`
   background: var(--Linkbrary-red, #ff5b56);
   font-family: Pretendard;
   font-size: 16px;
-  font-style: normal;
   font-weight: 600;
-  line-height: normal;
 `;
+interface Props {
+  setModalOpen: (id: { id: number | null }) => void;
```

Comment:
자세히 보지 않으면 id가 중첩되어 있는 것으로 오해하기 쉽겠네요,  조금 더 알아보기 쉽게 수정해보는건 어떨까요?  
```suggestion
  setModalOpen: (modalInfo: { id: number | null }) => void;
```